x-pack/auditbeat/module/system/sockets/_meta/data.json

```diff
@@ -0,0 +1,40 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "metricset": {
+        "module": "system",
+        "name": "sockets",
+        "rtt": 115
+    },
+    "system": {
+        "sockets": {
+            "socket": [
+                {
+                    "direction": "incoming",
+                    "family": "ipv4",
+                    "local": {
+                        "ip": "10.0.2.15",
+                        "port": 22
+                    },
+                    "process": {
```

Comment:
Please use the ECS field names for "process" and "user" as well

Reply:
Should be there now.